include/script_api/macros.h

```diff
@@ -571,7 +571,7 @@
 
 /****** COMMON SCRIPTS ************************************************************************************************/
 
-#define EXIT_WALK_SCRIPT(walkDistance, exitIdx, map, entryIdx) \
+#define EVT_EXIT_WALK(walkDistance, exitIdx, map, entryIdx) \
```

Comment:
I think these macros should have some kind of prefix to show that they produce a whole script rather than just one evt command as in EVT_WAIT()
e.g. `EVTS_EXIT_WALK`

Reply:
Yeah perhaps. For hud scripts I use HES_TEMPLATE_XXX, so I could an analogous EVT_TEMPLATE_YYY. We'll have to see how many of these we actually end up making.